src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -462,6 +474,10 @@ export default class SourceFileLinter {
 		return symbol.valueDeclaration?.getSourceFile().fileName === "/types/@ui5/linter/overrides/jquery.sap.d.ts";
 	}
 
+	isSymbolOfPseudoType(symbol: ts.Symbol | undefined) {
+		return symbol?.valueDeclaration?.getSourceFile().fileName.includes("/types/@ui5/linter/overrides/library/");
```

Comment:
Shouldn't this be `startsWith` instead of `includes`?

Reply:
Done